src/main/java/org/spongepowered/api/item/inventory/custom/CustomInventory.java

```diff
@@ -25,52 +25,212 @@
 package org.spongepowered.api.item.inventory.custom;
 
 import org.spongepowered.api.Sponge;
-import org.spongepowered.api.item.inventory.type.OrderedInventory;
-import org.spongepowered.api.text.translation.Translation;
+import org.spongepowered.api.item.inventory.Inventory;
+import org.spongepowered.api.item.inventory.crafting.CraftingInventory;
+import org.spongepowered.api.item.inventory.type.ChestInventory;
+import org.spongepowered.api.item.inventory.type.DispenserInventory;
+import org.spongepowered.api.text.Text;
 import org.spongepowered.api.util.Identifiable;
 import org.spongepowered.api.util.ResettableBuilder;
 
 /**
  * A CustomInventory is an inventory that has no backing in the world that is
  * created by plugins.
  */
-public interface CustomInventory extends OrderedInventory, Identifiable {
+public interface CustomInventory extends Inventory, Identifiable {
```

Comment:
You could literally rename CustomInventory to TitledInventory or something like that lol.